packages/snaps-execution-environments/lavamoat/browserify/policy-override.json

```diff
@@ -1,9 +1,3 @@
 {
-  "resources": {
-    "external:../snaps-utils/src/logging.ts": {
-      "globals": {
-        "console": true
-      }
-    }
-  }
+  "resources": {}
```

Comment:
Do we still need this file?

Reply:
We still have a few entries in it on the main branch. To remove it we would have to modify the build script on the audit branch which I think is an unnecessary deviation.